docs/evaluate/development-production-features/multi-tenant.mdx

```diff
@@ -0,0 +1,39 @@
+---
+id: multi-tenant
+title: Multi-tenant - Temporal feature
```

Comment:
I think "Multi-tenancy" might be a better name for the actual feature, but if this has already been decided, ignore me on this one